src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -44,6 +48,14 @@ const CpsRecommendations = ({ items, parentColumns }) => {
     translations,
   );
 
+  const { text, endTextVisuallyHidden } = path(['skipLink'], recommendations);
+
+  const skipLinkTerms = {
+    '%title%': title,
+  };
+
+  const skipEndId = 'recommendation-end';
```

Comment:
![Screenshot 2020-07-24 at 11 57 04](https://user-images.githubusercontent.com/1430787/88384953-51915b80-cda5-11ea-958d-a83ce1013b62.png)

We should probably write this with users in mind. Please can we call it `end-of-recommendations`?